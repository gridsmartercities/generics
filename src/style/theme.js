const theme = {
  main: {},
  input: {},
  label: {},
  buttons: {},
  mast: {},
  containers: {}
};

export default theme;
