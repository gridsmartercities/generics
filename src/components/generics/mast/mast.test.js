import "jest-dom/extend-expect";
import React from "react";
import { render } from "@testing-library/react";
import Mast from "./index";
import theme from "../../../style/theme";
import { ThemeProvider } from "styled-components";

test("Mast Component", () => {
  const { getByLabelText, container } = render(
    <ThemeProvider theme={theme}>
      <Mast onSubmit={() => "yo"}>
        <img src="image.jpg" alt="dogs" />
      </Mast>
    </ThemeProvider>
  );

  expect(container.firstChild).toMatchSnapshot();
});
