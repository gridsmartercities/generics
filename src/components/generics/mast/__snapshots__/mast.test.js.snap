// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mast Component 1`] = `
<section
  class="sc-bdVaJa cUTrPo"
>
  <img
    alt="dogs"
    src="image.jpg"
  />
</section>
`;
